src/js/common/schemaform/pages/applicantInformation.jsx

```diff
@@ -53,7 +53,7 @@ export default function applicantInformation(schema, options) {
     initialData: {},
     uiSchema: _.assign({
       'ui:order': fields,
-      'ui:description': <p>You aren’t required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>,
+      'ui:description': <p>You aren’t required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>,
```

Comment:
Should we use the ApplicantDescription component here?

Reply:
Thanks @jbalboni, excellent idea, change made.